mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -1248,6 +1263,11 @@ class LogisticRegressionModel private[spark] (
     }
   }
 
+  private[spark] override def estimatedSize: Long = {
+    SizeEstimator.estimate((this.params, this.uid,
```

Comment:
NIT: Can we just make the SizeEstimator take varargs instead of creating a tuple here.